.circleci/config.yml

```diff
@@ -820,6 +831,12 @@ workflows:
       - build_docs
       - black
       - flake8
+      - deploy_reliability:
+          filters:
+            branches:
+              only:
+                - master
+                - /.*-reliability/
```

Comment:
should we just do like `master` and `staging`?

Reply:
Thought here was that I was hoping to be able to push my performance testing changes to a reliability branch to be able to make verifying easier. 🙂